tests/dda/upload/test_wrapper.py

```diff
@@ -1,26 +1,101 @@
 import os
+import tempfile
 import uuid
 
 import pytest
 
 from dagshub.upload import Repo
+from tests.dda.mock_api import MockApi
 
 
 @pytest.fixture(scope="function")
-def test_file():
+def test_file() -> str:
     filepath = f"{uuid.uuid4()}.txt"
     with open(filepath, "w") as f:
         f.write("test data")
     yield filepath
     try:
         os.remove(filepath)
-    except OSError:
+    except (OSError, FileNotFoundError):
         pass
 
 
-def test_upload_dataset_closes_files(mock_api, upload_repo: Repo, test_file: str):
+@pytest.fixture(scope="function")
```

Comment:
Default scope is function
```suggestion
@pytest.fixture
```